### PR TITLE
Update results

### PR DESCRIPTION
**Pull request by asalvi0**
_Re-open cleaner PR._

### CODE REVIEW

File: criterion.toml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+#   presenting output numbers, eg. on a library's README.
+# verbose: Like criterion, but prints additional statistics.
+# bencher: Emulates the output format of the bencher crate and nightly-only
+#   libtest benchmarks.
+output_format = "quiet"
+
+# This is used to configure the plotting backend used by cargo-criterion.
+# Options are "gnuplot" and "plotters", or "auto", which will use gnuplot if it's
+# available or plotters if it isn't.
+ploting_backend = "auto"
```
Comment (djc) [inline criterion.toml:19]: `ploting_backend` appears to be misspelled. Do we actually need this? Why?

File: Cargo.toml [hunk 2]
```diff
@@ -3,11 +3,11 @@ name = "template-benchmarks-rs"
 version = "0.1.0"
 authors = ["Dirkjan Ochtman <dirkjan@ochtman.nl>"]
 build = "src/build.rs"
-edition = "2018"
+edition = "2021"
 
 [dependencies]
 askama = { git = "https://github.com/djc/askama", branch = "main" }
-criterion = "0.4"
+criterion = { version = "0.4", features = ["html_reports"] }
```
Comment (djc) [inline Cargo.toml:10]: Why enable the HTML reports here?

Comment (asalvi0) [reply Cargo.toml:10]: Added because it was not generating the svg output even if _(gnuplot was installed)_.

**Review by djc — APPROVED** [review]

**Comment by djc** [comment]
Thanks!

**Pull request merged**